tools/JsonCli/.vscode/tasks.json

```diff
@@ -12,18 +12,36 @@
             "problemMatcher": "$msCompile"
         },
         {
-            "label": "publish",
+            "label": "publish2",
```

Comment:
I don't really like these labels as they seme more numerical rather than framework-specific.  What do you think about publish (2.0) and publish (3.0) or something like that?

Reply:
No preference - this is an internal tool that we touch like once a year haha